README.md

```diff
@@ -72,6 +72,8 @@ model = WhisperModel(model_size, device="cuda", compute_type="float16")
 # model = WhisperModel(model_size, device="cuda", compute_type="int8_float16")
 # or run on CPU with INT8
 # model = WhisperModel(model_size, device="cpu", compute_type="int8")
+# or use available models (https://huggingface.co/models?search=faster-whisper) from Huggingface
+# model = WhisperModel('zh-plus/faster-whisper-large-v2-japanese-5k-steps')
```

Comment:
I think this comment is not useful here. It's better to keep everything in the conversion section. Can you remove these lines?

Reply:
Done.